.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,30 @@
+name: build-and-test
+on: [pull_request]
+jobs:
+  test-re2:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Checkout submodules
+        run: git submodule update --init --recursive
+      - name: Install arrow
+        run: |
+          sudo apt update
+          sudo apt install -y -V ca-certificates lsb-release wget
+          wget https://apache.bintray.com/arrow/$(lsb_release --id --short | tr 'A-Z' 'a-z')/apache-arrow-archive-keyring-latest-$(lsb_release --codename --short).deb
+          sudo apt install -y -V ./apache-arrow-archive-keyring-latest-$(lsb_release --codename --short).deb
+          sudo apt update
+          sudo apt install -y -V libarrow-dev
```

Comment:
Make sure to install a specific arrow version. Example [here](https://github.com/abs-tudelft/fletcher/blob/0.0.19/.github/workflows/test.yml#L9-L10) and [here](https://github.com/abs-tudelft/fletcher/blob/0.0.19/.github/workflows/test.yml#L30). If a new Arrow version is released your builds will start failing (because you request a specific version in the CMake file).